### ISSUE TITLE
[Proposal] for templates in create-yoshi-app use single bracket configuration

### ISSUE THREAD

**Issue by sidoruk-sv** [wix/yoshi#2328]
# 🙋 Feature Request

### 😯 Current Behavior

For now, i18next is configured by default so variables inside keys will be declared like 
```
"key_name": "value with {{variable}}"
``` 

If anytime in the future the project will need to switch to ICU support, it will be needed to change `interpolation` option to single brackets:
with such config
```
{
...
interpolation: {
  prefix: '{',
  suffix: '}'
},
...
}
```
keys need to be changed to:
```
"key_name": "value with {variable}"
```

Explained in `wix-i18n-config` [readme](https://github.com/wix-private/fed-infra/tree/master/wix-i18n-config#q--a):
> Q: I want to migrate an existing project to use wix-i18n-config, is there something that I need to change with the translations?
>
>  A: You have to make sure that all of the formatting and interpolations are using single the curly brackets format, which is defined by ICU. For example, if you have a translation key with this value: "Click this link: {{link}}" you should manually transform it in all messages_xx.json files to look like this "Click this link: {link}". Also note, that if the parameter that you're interpolating is a number or a date you should leverage the ICU formatting (e.g. "Your apple count is: {appleCount, number}". You can find more formatting examples here

This change will lead to re-translating all keys inside the project (cause, for now, there is no easy migration from `{{` to `{` in Smartling).

As heard here: 
https://wix.slack.com/archives/CER7YC83G/p1588597109152800?thread_ts=1588595602.151100&cid=CER7YC83G

### 🤔 Expected Behavior

No need to re-translate the whole project

### 💁 Possible Solution 

Use "single brackets" config by default for all templates.

### List of templates need to be changed (both javascript and typescript):
- [x] fullstack;
- [x] client;
- [x] out-of-iframe;
- [ ] business-manager-module;

What do you think?

**Comment by ranyitz** [comment]
That makes sense to me @amitdahan WDYT?

**Comment by amitdahan** [comment]
Sounds good.
So templates will have the new brackets which will be a first step towards using translations the same way it's done in `wix-i18n-config`?

**Comment by sidoruk-sv** [comment]
yes, and there will be no need of "breaking step" in the future

**Comment by amitdahan** [comment]
Sounds good!

**Comment by sidoruk-sv** [comment]
`business-manager-module` case will be resolved when #2363 will be merged

**Comment by ranyitz** [comment]
#2363 has been merged
